CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
 ## 0.9.2 (Unreleased)
```

Comment:
Please remove this part. We'll do it when we cut a new release.

Reply:
Sure, sounds good!